db/migrate/20230404130043_add_secrets_image_to_organization.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddSecretsImageToOrganization < ActiveRecord::Migration[6.1]
```

Comment:
We should allow reversibility of this migration by adding columns in `up` method and removing columns in `down` method